docs/sql-programming-guide.md

```diff
@@ -1253,7 +1253,12 @@ This command builds a new assembly jar that includes Hive. Note that this Hive a
 on all of the worker nodes, as they will need access to the Hive serialization and deserialization libraries
 (SerDes) in order to access data stored in Hive.
 
-Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`.
+Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`. Please note when running 
```

Comment:
I meant `hive-site.xml` specifically? 
The Java 6 comment is out of date or will be soon I believe.